ideascube/monitoring/tests/test_views.py

```diff
@@ -216,6 +226,16 @@ def test_staff_can_create_specimen_with_letters_and_ints_in_barcode(staffapp):
     assert item.specimens.filter(barcode='123abc').count()
 
 
+def test_staff_can_create_specimen_with_letters_and_ints_in_serial(staffapp):
```

Comment:
> I see no reason why it couldn't.

Me neither, but that might just be that you and I aren't aware of how things are expected to work in the field.

Currently, without this test, then we just don't know what the serial string can contain (which probably means it can contain anything). Adding this test enshrines a given format, documents it as the expected norm.

Basically, since we don't know that this is the expected norm, then we shouldn't add this test, because it might be wrong.

However we should try to obtain details on how serials are actually formed, and validate **that** with a test. (along with the required validation on the form side of things)

But that would probably be a concern for a very different pull request.